app/components/review_tasks/after_sign_off_component.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+module ReviewTasks
+  class AfterSignOffComponent < ViewComponent::Base
+    def initialize(planning_application:)
+      @planning_application = planning_application
+      @recommendation = planning_application.recommendation
+    end
+
+    private
+
+    def render?
+      @planning_application.recommendation_review_complete? && challenged?
+    end
+
+    def challenged?
```

Comment:
Ugly code. It shouldn't need this protection but there are many tests that have not set a recommendation when accessing the review task list code. Maybe it's "better protected" but I would have preferred it without all this protective gubbins.

Reply:
Haha, I hit the same issue yesterday!